src/pages/Build.vue

```diff
@@ -209,6 +217,22 @@
         </q-card-actions>
       </q-card>
     </q-dialog>
+    <q-dialog v-model="delete_build">
+      <q-card style="width: 400px;">
+        <q-card-section>
+          <div class="text-h6">Warning</div>
+        </q-card-section>
+        <q-card-section>
+          You are going to delete {{build.id}} build, are you sure ?
```

Comment:
it might be worth swapping the words build and {{build.id}} in places, because user may think that they are deleting several builds at once )

Reply:
Fixed